app/web/features/mod/hooks.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  modUserDetailsKey,
+  modUserKey,
+  newUsersListKey,
+} from "features/queryKeys";
+import { userStaleTime } from "features/userQueries/constants";
+import { RpcError } from "grpc-web";
+import { ListUserIdsRes, UserDetails } from "proto/admin_pb";
+import { User } from "proto/api_pb";
+import { useInfiniteQuery, useQuery } from "react-query";
+import { service } from "service";
+
+export const useNewUsers = () => {
+  const query = useInfiniteQuery<ListUserIdsRes.AsObject, RpcError>(
+    newUsersListKey,
+    ({ pageParam }) =>
+      service.admin.listUserIds({
+        startTime: new Date(1970, 0, 0, 0, 0, 1),
+        endTime: new Date(2050, 0, 0),
+        pageSize: 50,
+        pageToken: pageParam,
+      }),
+    {
+      getNextPageParam: (lastPage) => lastPage.nextPageToken || undefined,
+      refetchInterval: 60_000,
```

Comment:
Should it have the underscore? The docs say the type is just `number` and that doesn't seem very javascripty?

Reply:
It's just for clarity, it's called "numeric separators" I think